rpcs3/Emu/Cell/lv2/sys_time.h

```diff
@@ -6,6 +6,6 @@
 // SysCalls
 
 error_code sys_time_get_timezone(vm::ptr<s32> timezone, vm::ptr<s32> summertime);
-error_code sys_time_get_current_time(vm::ptr<s64> sec, vm::ptr<s64> nsec);
+error_code sys_time_get_current_time(vm::ptr<u64> sec, vm::ptr<u64> nsec);
```

Comment:
wtf.
sys_time_sec_t and sys_time_nsec_t are defined as int64_t

Reply:
They are? I did some googling and wasn't able to find confirmation of this.